deployments/knack_banner.yaml

```diff
@@ -0,0 +1,56 @@
+###
+### A complete description of a Prefect Deployment for flow 'Knack HR Banner'
+###
+name: HR Knack Banner
+description: null
+version: 7167d8ebe8099fb6d815d255df90f5d9
+# The work queue that will handle this deployment's runs
+work_queue_name: atd-data-03
+work_pool_name: default-agent-pool
+tags:
+- production
+parameters: {}
+schedule:
+  cron: 45 13 * * *
+  timezone: null
+  day_or: true
+is_schedule_active: true
+infra_overrides: {}
+
+###
+### DO NOT EDIT BELOW THIS LINE
+###
+flow_name: Knack HR Banner
+manifest_path: null
+infrastructure:
+  type: process
+  env: {}
+  labels: {}
+  name: null
+  command: null
+  stream_output: true
+  working_dir: null
+  _block_document_id: 0c387fef-70e3-4193-aea3-22791b1f6e3f
+  _block_document_name: anonymous-bd583a80-cb23-4ec5-9c1a-8a219122502a
+  _is_anonymous: true
+  block_type_slug: process
+  _block_type_slug: process
+storage:
+  repository: https://github.com/cityofaustin/atd-prefect/
+  reference: main
+  access_token: null
+  include_git_objects: false
+  _block_document_id: d0cc8e56-412b-49b3-bb9c-33b522c64736
+  _block_document_name: atd-prefect-main-branch
```

Comment:
i'm worried about the management of all these blocks becoming an overwhelming mess.
<img width="1089" alt="Screen Shot 2023-02-23 at 10 59 15 AM" src="https://user-images.githubusercontent.com/14793120/220961960-5c586929-d61d-4a4f-bc3f-0590408ea579.png">

Reply:
We could potentially offload our secret storage to S3 or our password manager of choice?

Reply:
Is there a known limit on the length of the secret's value in a secret block? I don't see anything jumping out at me from the docs, so I'm inclined to want to use those to house our secrets as they are obfuscated in the prefect UI. 

I think it would be pretty clean to have a single secret per flow, and have that secret be a JSON blob with key-value pairs as needed, or even a more complex shape if the flow warrants it. 

I'm not against using S3, but I'd want to warn against it because that pushes a lot of responsibility on us to make sure that resource is properly secured from the public (or even non-DTS folks) yet remains available to flows as needed. We'd have to have a secret to give us access to our other secrets, which doesn't feel awesome.

I think it would be really slick if we could use 1Password as the secret store (it's like its true raison d'etre, if anything), but I'd also want to see it's reliability proven up with a discovery day experiment or something along those lines.

Reply:
I like the idea of a central secrets store like 1Password like Frank mentioned. If that doesn't pan out, we could always looks into AWS Secrets Manager as a central source of secrets. Some of these flows could add up to a lot of API calls over time and get $$$, though. 

I'm trying out the Secret blocks for the atd-service-bot flows to store an encoded JSON string and then load it in the script. One downside that I see to this is that we'll probably need to document the expected fields in the JSON or it might be hard to modify/create since you can't retrieve the value once it is set in the Prefect UI (which is a good thing for security).